src/services/user.js

```diff
@@ -34,7 +34,7 @@ import storage from './storage';
  * Get events attended by the user
  * @param {String} username username of the user
  * @param {String} termCode code for the semester
- * @return {Promise.<attendedEvents[]>} An object of all CL&W events attended by the user
+ * @return {Promise.<AttendedEvents[]>} An object of all CL&W events attended by the user
```

Comment:
This should be `Promise.<AttendedEvent[]>` (drop the "s" on `AttendedEvents`) because that's the name of the `@typedef`.